modules/hangman.js

```diff
@@ -6,134 +6,233 @@ const log = utils.getLogger('hangman');
-            ' _|________```\n' +
+            ' _|________\n```\n' +
             'Use :regional_indicator_a::regional_indicator_b::regional_indicator_c: ... :regional_indicator_z: ' +
             'reactions to pick letters.';
 
         // instantiate embed object
         const embed = new Discord.MessageEmbed({
-            author: {name: "Guess the card:"},
+            author: {name: 'Guess the card:'},
             title,
             description,
-            footer: {text: "You have "+this.gameTime/60000+" minutes to guess the card."}
+            footer: {text: 'You have ' + GAME_TIME / 60000 + ' minutes to guess the card.'}
         });
 
         // game is over
-        if (done || !missing.length || wrong > 6) {
-            embed.setTitle(card.name);
+        if (this.done || !missing.length || wrong > 6) {
+            embed.setTitle(this.card.name);
             embed.setFooter(missing.length ? 'You failed to guess the card!':'You guessed the card!');
-            embed.setURL(card.scryfall_uri);
-            if (card.layout === 'transform' && card.card_faces && card.card_faces[0].image_uris) {
-                embed.setImage(card.card_faces[0].image_uris.normal);
-            } else if(card.image_uris) {
-                embed.setImage(card.image_uris.normal);
+            embed.setURL(this.card.scryfall_uri);
+            if (this.card.layout === 'transform' && this.card.card_faces && this.card.card_faces[0].image_uris) {
+                embed.setImage(this.card.card_faces[0].image_uris.normal);
+            } else if(this.card.image_uris) {
+                embed.setImage(this.card.image_uris.normal);
             }
             embed.setColor(missing.length ? 0xff0000 : 0x00ff00);
         }
 
         return embed;
     }
+}
+
+class MtgHangman {
+    constructor() {
+        this.commands = {
+            hangman: {
+                aliases: [],
+                inline: false,
+                description: 'Start a game of hangman, where you have to guess the card name with reaction letters',
+                help: 'Selects a random Magic card, token oder plane and shows you the ' +
+                'placeholders for each letter in its name. To guess a letter, add a reaction to the ' +
+                'Hangman-message of the bot. Reactions have to be selected among the regional indicators ' +
+                ':regional_indicator_a: to :regional_indicator_z:. You can only have one active game of ' +
+                'Hangman per Discord server, but you can also start an additional one in a private query.\n\n' +
+                '**Difficulty**:\n' +
+                'With an optional parameter you can select the difficulty. Available are `easy`, `medium` ' +
+                'and `hard`. Default is medium. "Easy" includes the mana cost and type line, "Medium" includes ' +
+                'the mana cost and "Hard" doesn\'t include any hints.',
+                examples: ["!hangman", "!hangman easy"]
```

Comment:
Something like this:
```suggestion
                'the mana cost and "Hard" doesn\'t include any hints.\n\n'+
                '**Direct guessing**\n' + 
                'You can try to directly guess the card with `!hangman guess <cardname>`, but guessing the wrong '+
                'card will add another strike to the game in progress.',
                examples: ["!hangman", "!hangman easy", "!hangman guess wasteland"]
```